_pages/about.md

```diff
@@ -2,7 +2,9 @@
 layout: about
 title: about
 permalink: /
-description: <a href="#">Affiliations</a>. Address. Contacts. Moto. Etc.
+description: This is a description that is used in meta tags and will show up in search engine results.
+
```

Comment:
```suggestion
```
let's remove description from the about page and to make it intentionally fall back to `site.description`. you already have the fall back implemented in `_includes/metadata.html`: 
https://github.com/alshedivat/al-folio/blob/964b0eb741054614d5f9cd3cef4fea3532bb44a3/_includes/metadata.html#L28